### PR TITLE
Remove addEvents from examples

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "eslint-plugin-tree-shaking": "^1.7.3",
     "gl": "^4.2.2",
     "mkdirp": "^0.5.1",
+    "mjolnir.js": "^2.1.2",
```
Comment (ibgreen) [inline package.json]: Why would we have mjolnir.js in the main package.json? Shouldn't this be an optional dependency for examples.

File: yarn.lock [hunk 2]
```diff
@@ -4493,6 +4493,11 @@ gzip-size@^5.0.0:
     duplexer "^0.1.1"
     pify "^4.0.1"
 
+hammerjs@^2.0.8:
```
Comment (ibgreen) [inline yarn.lock]: Related to above, do these need to appear in the master yarn.lock?

File: examples/lessons/11/app.js [hunk 3]
```diff
@@ -77,7 +77,8 @@ export default class AppAnimationLoop extends AnimationLoop {
   }
 
   onInitialize({canvas, gl}) {
-    addMouseHandler(canvas);
+    const eventManager = new EventManager(canvas);
```
Comment (ibgreen) [inline examples/lessons/11/app.js]: One line comment explaining why we went with mjolnir (as opposed to built in event handling?)

File: examples/lessons/10/app.js [hunk 4]
```diff
@@ -48,8 +48,9 @@ export default class AppAnimationLoop extends AnimationLoop {
   }
 
   onInitialize({canvas, gl}) {
-    addKeyboardHandler(canvas, currentlyPressedKeys);
-    addMouseHandler(canvas);
+    const eventManager = new EventManager(canvas);
```
Comment (ibgreen) [inline examples/lessons/10/app.js]: One line comment explaining why this example went with mjolnir (as opposed to built in event handling?). 

Imagining a user new to luma.gl who has never heard about mjolnir.js (which is not mention ed in the luma docs) reading this code for the first time...

File: package.json (hunk 1 above)
Comment (Pessimistress) [reply package.json]: We load the examples in the browser render tests. They would crash if mjolnir is not found from the test directory.

**Pull request merged**